### PR TITLE
GKE single cluster configuration archetype

### PR DESCRIPTION
**Pull request by kumare3**
Demo purposes. To make it work, users will need to change some config

### CODE REVIEW

File: deployment/gke-single-cluster/flyte_generated.yaml [hunk 1]
```diff
@@ -0,0 +1,1550 @@
...
+      metrics-scope: "datacatalog"
+      profiler-port: 10254
+    application:
+      grpcPort: 8089
+    storage:
+      connection:
+        access-key: minio
+        auth-type: accesskey
+        disable-ssl: true
+        endpoint: http://minio.flyte.svc.cluster.local:9000
```
Comment (wild-endeavor) [inline deployment/gke-single-cluster/flyte_generated.yaml]: we're going to stick with using Minio for this?  should we move to gcs?

File: deployment/gke-single-cluster/flyte_generated.yaml [hunk 2]
```diff
@@ -0,0 +1,1550 @@
...
+      - id: staging
+        name: staging
+      - id: production
+        name: production
+      - id: domain
+        name: domain
+    registration:
+      maxWorkflowNodes: 100
+    scheduler:
+      eventScheduler:
```
Comment (wild-endeavor) [inline deployment/gke-single-cluster/flyte_generated.yaml]: should we remove this section completely or is there a gcp equivalent we can move to?

File: deployment/gke-single-cluster/flyte_generated.yaml [hunk 3]
```diff
@@ -0,0 +1,1550 @@
...
+        app: flyteadmin
+    spec:
+      containers:
+      - command:
+        - flyteadmin
+        - --logtostderr
+        - --config
+        - /etc/flyte/config/flyteadmin_config.yaml
+        - serve
+        image: docker.io/lyft/flyteadmin:v0.1.4
```
Comment (wild-endeavor) [inline deployment/gke-single-cluster/flyte_generated.yaml]: this says 0.1.4, but the overlay file has 0.1.1.  I'll remove that at some point.  no reason to specify if it's not being used.

File: deployment/gke-single-cluster/flyte_generated.yaml (hunk 1 above)
Comment (ChethanUK) [reply deployment/gke-single-cluster/flyte_generated.yaml]: > we move to gcs?

Moving to GCS will be good..

File: kustomize/overlays/gke-single-cluster/propeller/propeller.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+metadata:
+  name: flytepropeller
+  namespace: flyte
+spec:
+  template:
+    spec:
+      containers:
+      - name: flytepropeller
+        env:
+        - name: QUBOLE_API_KEY
```
Comment (chethanu) [inline kustomize/overlays/gke-single-cluster/propeller/propeller.yaml]: What happen if we don't use QUBOLE?

**Comment by mweagle** [comment]
Is this still valid @kumare3 ?

**Pull request closed without merging**